tests/components/sensor/test_rest.py

```diff
@@ -24,13 +25,6 @@ def tearDown(self):
         """Stop everything that was started."""
         self.hass.stop()
 
-    def test_setup_missing_config(self):
```

Comment:
As this depends on the validation schema of the platform, I think it's good to test this for every platform.